test/extensions/voting-rep.js

```diff
@@ -1788,7 +1789,7 @@ contract("Voting Reputation", (accounts) => {
       summary = await voting.getActionSummary(NO_ACTION, ADDRESS_ZERO);
       expect(summary.sig).to.equal(NO_ACTION);
       expect(summary.expenditureId).to.be.zero;
-      expect(summary.domainSkillId).to.be.zero;
+      // expect(summary.domainSkillId).to.be.zero; // We leave this undefined
```

Comment:
By 'undefined' here do you mean it could have an arbitrary value depending on what other actions are in a possible multicall? It certainly does have a value here and could be tested even if that is the case (as the setup is deterministic).

Somewhat relatedly, I'm a little worried about the change in behaviour in `getActionSummary`, where previously if we hit `NO_ACTION` or `OLD_MOVE_FUNDS` we overwrote any previous skill or domain ids we had seen, but now we just set the signature and continue to iterate. I've not been able to come up with a scenario where something breaks in VotingReputation as a result, but am not at the top of my game today and would appreciate you taking a look as well if you haven't already.

Reply:
What I mean by "undefined" is that the spec does not specify a result. So while in this (deterministic) call the result is known, with different parameters the result may be different. So rather than test this particular result, which I suggest could be misleading, I would rather leave the result un-tested.

Reply:
Regarding the change in `getActionSummary`, why not simply preserve the existing behavior? It's may be a bit of wasted gas, but not the end of the world. Alternatively, we could take my argument of "undefined results" and extend it here.